composer.json

```diff
@@ -10,7 +10,9 @@
         { "name": "Jérôme Tamarelle", "email": "jerome.tamarelle@mongodb.com" }
     ],
     "require": {
-        "php": "^8.1"
+        "php": "^8.1",
```

Comment:
Why does the generator use `>=8.1` instead of `^8.1`? Should these be consistent?

Reply:
I always go with `>=8.1` in open source projects, as I don't know the compatibility of new versions yet and I don't want to block users. As this is a repo by a big company this problem should not occur 😄

Reply:
Fixed.